src/App.tsx

```diff
@@ -1,223 +1,75 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
 import { AuthContext } from './components/Auth/AuthContext';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoFooter } from './components/TodoFooter/TodoFooter';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { LoadingError } from './components/LoadingError/LoadingError';
 
 export const App: React.FC = () => {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
 
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterTodo, setFilterTodo] = useState('all');
```

Comment:
Better crate enum for filter options. When you would write test tasks, using `enum` in simular case would be a ➕